packages/fxa-auth-server/lib/senders/emails/templates/cadReminderSecond/index.stories.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+import * as cadReminderFirst from '../cadReminderFirst/index.stories';
```

Comment:
We don't need to import everything (`*`) out of this template. If you change [this line](https://github.com/mozilla/fxa/blob/f6e682e795b9c3a537574fb8aff8f18921a98c54/packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/index.stories.ts#L17) to `export defaultVariables = {`, you can use `import { defaultVariables as cadReminderDefaultVariables } from '../cadReminderFirst/index.stories'`. (or rename `defaultVariables` in that file to use `cadReminder` so you don't have to rename it here)

Reply:
I think initially I tried to import the args this way but since they are getting exported from a stories file, storybook is treated it as a story and threw errors
![image](https://user-images.githubusercontent.com/37609519/129389797-7c87657e-80ae-43c6-a8b8-2e539e0d7de3.png)

Reply:
Oooh interesting! We could create an additional "shared" file for stuff like this but eh since it's just for storybook, we'll only have shared args like this between emails that have first version/second etc. and you're doing what the docs recommend I say let's leave it like how you have it for now👍